bertopic/_bertopic.py

```diff
@@ -497,10 +497,14 @@ def transform(self,
         ```
         """
         check_is_fitted(self)
-        check_embeddings_shape(embeddings, documents)
-
+        
         if isinstance(documents, str) or documents is None:
             documents = [documents]
+        
+        if len(documents) == 1 and isinstance(embeddings, np.ndarray) and embeddings.ndim == 1:
+            embeddings = embeddings.reshape(1,-1)
```

Comment:
The `embeddings` that you can pass to `.transform` is typed as `np.ndarray` which is actually quite misleading (sorry!) as it can technically take the form of any iterable (but mostly just np.ndarray-like structures, such as a scipy sparse matrix). As such, doing something like `embeddings.ndim == 1` might break here.